src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File firstFile = new File(fromFileName);
+        File secondFile = new File(toFileName);
+
+        int buy = 0;
+        int supply = 0;
+        int result = 0;
+
+        try {
+            List<String> stringList = Files.readAllLines(firstFile.toPath());
+            for (String s : stringList) {
+                if (s.contains("buy")) {
+                    buy += Integer.parseInt(s.substring(s.indexOf(',') + 1, s.length()));
```

Comment:
Using `substring` method makes your code less readable. Consider using `split` method instead. It's a more straightforward way to separate a string into parts.